python/rikai/types/vision.py

```diff
@@ -37,15 +37,41 @@ def __init__(self, uri: str):
         super().__init__(uri)
         self._cached_data = None
 
+    def show(self, **kwargs):
```

Comment:
Hmm, do we expect `show()` to be implemented by all the object?  a little bit worried that this makes the jupyter integration not transparent.

Reply:
the default _repr_*_ makes it transparent if you don't need additional display options. The `show` just allows for customization

Reply:
should we rename `show()` to `display` ? it is consistent with jupyter's naming conventions?

It is like
```
#(inexplicably) 
display(video) 
==>
video.display(foo, bar)
```